pkg/controllers/dynakube/extension/utils/naming.go

```diff
@@ -0,0 +1,18 @@
+package utils
+
+import (
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
+	"github.com/Dynatrace/dynatrace-operator/pkg/controllers/dynakube/extension/consts"
+)
+
+func BuildPortsName() string {
```

Comment:
```suggestion
func BuildPortName() string {
```

Reply:
fixed in 6c108872af9035d9124d0ac5c6bc4565b5ae7c74